flink-runtime/src/main/java/org/apache/flink/runtime/execution/Environment.java

```diff
@@ -258,4 +259,10 @@ default void setAsyncOperationsThreadPool(ExecutorService executorService) {}
     default ExecutorService getAsyncOperationsThreadPool() {
         throw new UnsupportedOperationException();
     }
+
+    default void setCheckpointStorageAccess(CheckpointStorageAccess checkpointStorageAccess) {}
+
+    default CheckpointStorageAccess getCheckpointStorageAccess() {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
I think this (and related) change belong to the wrong commit.

Reply:
> Thanks updating the PR @curcur.
> 
> I've left some comments, PTAL.
> 
> Besides that, could you please name the extracted commits according to [conventions](https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html#commit-naming-conventions) (add `[hotfix]` and component label, fix case, etc.).

which of the commits do you think not following the convention?

If you mean the [hotfix] one, It is to fix a file naming issue for log4j2 file. I am not sure whether it is belonging to a "component"